openquake/calculators/scenario_damage.py

```diff
@@ -89,28 +88,43 @@ def scenario_damage(riskinputs, riskmodel, rlzs_assoc, monitor):
     :param monitor:
         :class:`openquake.baselib.performance.PerformanceMonitor` instance
     :returns:
-        a dictionary {('asset', asset): <mean stddev>,
-                      ('taxonomy', asset.taxonomy): <damage array>}
+        a dictionary {'asset': [(l, r, a, mean-stddev), ...],
+                      'taxonomy': damage array of shape T, L, R, E, D}
```

Comment:
This should be updated, since the new result dictionary has two additional keys.